### PR TITLE
[ETHEREUM-CONTRACTS] Fix SuperTokenV1Library hash

### PR DESCRIPTION
**Pull request by 0xdavinchee**
keccak256("org.superfluid-finance.agreements.InstantDistributionAgreement.v1") is `0x8aedc3b5d4bf031e11a7e2940f7251c005698405d58e02e1c247fed3b1b3a674`, not `0x15609310ae3c30189a1218b7adabaf36c267255e70cf91b6cba384367d9eda32`.

[Verify](https://emn178.github.io/online-tools/keccak_256.html): `org.superfluid-finance.agreements.InstantDistributionAgreement.v1`

### CODE REVIEW

File: packages/ethereum-contracts/test/contracts/apps/SuperTokenV1Library.IDA.test.ts [hunk 1]
```diff
@@ -288,12 +288,10 @@ describe("IDAv1Library testing", function () {
         });
 
         it("#1.7 - _getHostAndIDA empty cache test", async () => {
-            await expect(
-                superTokenLibIDAMock.listSubscriptionsTest(
-                    superToken.address,
-                    bob
-                )
-            ).to.be.reverted;
+            await superTokenLibIDAMock.listSubscriptionsTest(
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/test/contracts/apps/SuperTokenV1Library.IDA.test.ts:291]: this was reverting before because the hash was wrong

Comment (hellwolf) [reply packages/ethereum-contracts/test/contracts/apps/SuperTokenV1Library.IDA.test.ts:291]: okay

File: packages/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol [hunk 2]
```diff
@@ -1566,7 +1566,7 @@ library SuperTokenV1Library {
             }
             cfa = IConstantFlowAgreementV1(address(ISuperfluid(host).getAgreementClass(
                 //keccak256("org.superfluid-finance.agreements.ConstantFlowAgreement.v1")
-                    0xa9214cc96615e0085d3bb077758db69497dc2dce3b2b1e97bc93c3d18d83efd3)));
+                0xa9214cc96615e0085d3bb077758db69497dc2dce3b2b1e97bc93c3d18d83efd3)));
```
Comment (hellwolf) [inline packages/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol]: why hardcode the hashes? I would expect this being a trivial optimization compiler will figure out itself.

**Review by hellwolf — APPROVED** [review]

File: packages/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol (hunk 2 above)
Comment (0xdavinchee) [reply packages/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol]: verified that compiler optimization handles this 👍🏼

**Pull request merged**